src/esp32/ld/esp32.rom.libc-funcs.ld

```diff
@@ -0,0 +1,79 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
```

Comment:
Let's make it consistent with the requirements of this project

Reply:
For consistency, removed the license tags from the linker files.